client/devdocs/design/search-collection.jsx

```diff
@@ -64,7 +65,7 @@ const Collection = ( { children, filter, section = 'design', component } ) => {
 				unique={ !! component }
 				url={ exampleLink }
 			>
-				{ example }
+				{ component ? <PropsViewer component={ component } example={ example } /> : example }
```

Comment:
>This is intentional, such that the `PropsViewer` could change the props to the the `example` component.

Is that coming in a future pull request? I didn't see anything about it here. Maybe a bit premature on how we'd expect that form component to be rendered.